src/llm/agent-evals/slack-agent/test-data.ts

```diff
@@ -0,0 +1,270 @@
+        arguments: {
+          channel_id: 'C874HKJ',
+          text: 'We have a project kickoff meeting at 3pm. Please join if you can.'
+        }
+      }
+    ],
+    expected_response:
+      "Joined #project-x and posted: 'We have a project kickoff meeting at 3pm. Please join if you can.'",
+    tool_mock_response_overrides: {}
+  },
+  {
+    description: 'Reply to a thread in #project-x channel in the discussion about login issues.',
+    chat_history: [
+      { author: 'John_Snow', message: 'Can someone help with the login issue?' },
+      { author: 'Slack_Bot', message: 'Check the error logs in #project-x.' }
+    ],
+    invocation: {
+      initiator_name: 'John_Snow',
+      message:
+        "@Quix Reply 'Please provide more details' to the thread related to login issues in #project-x channel"
+    },
+    reference_tool_calls: [
+      {
+        name: 'slack_list_channels',
+        arguments: {
+          limit: 100,
+          cursor: ''
+        }
+      },
+      {
+        name: 'slack_get_channel_history',
+        arguments: {
+          channel_id: 'C874HKJ',
+          limit: 10
+        }
+      },
+      {
+        name: 'slack_reply_to_thread',
+        arguments: {
+          channel_id: 'C874HKJ',
+          thread_ts: '1716282200.000005',
+          text: 'Please provide more details'
+        }
+      }
+    ],
+    expected_response: "Replied in the thread in #project-x channel: 'Please provide more details'",
+    tool_mock_response_overrides: {}
+  },
+  {
+    description: 'Post about our new member Jamie in #general channel',
+    chat_history: [],
+    invocation: {
+      initiator_name: 'Jamie_Lannister',
+      message:
+        '@Quix Post about our new member Jamie getting all details from the user profile in #general channel'
+    },
+    reference_tool_calls: [
+      {
+        name: 'slack_get_users',
+        arguments: {
+          limit: 100
+        }
+      },
+      {
+        name: 'slack_get_user_profile',
+        arguments: {
+          user_id: 'U90JAMIW'
+        }
+      },
+      {
+        name: 'slack_list_channels',
+        arguments: {
+          limit: 100,
+          cursor: ''
+        }
+      },
+      {
+        name: 'slack_post_message',
+        arguments: {
+          channel_id: 'C134DSD',
+          text: 'Jamie is our new member. Welcome to the team!'
+        }
+      }
+    ],
+    expected_response:
+      "Posted in #general channel: 'Jamie is our new member. Welcome to the team!'",
+    tool_mock_response_overrides: {
+      slack_get_user_profile: {
+        user_id: 'U90JAMIW'
+      }
+    }
+  },
+  {
+    description: 'Leave #project-x channel',
+    chat_history: [
+      {
+        author: 'Jamie_Lannister',
+        message: '@Quix Leave #project-x channel'
+      },
+      {
+        author: 'Slack_Bot',
+        message:
+          'Are you sure you want to leave the #project-x channel? Please confirm by saying "Yes".'
+      }
```

Comment:
Aren't you expecting this message to be from quix bot? It should be something like the following
```suggestion
      {
        author: 'Quix (bot)',
        is_bot: true,
        message:
          'Are you sure you want to leave the #project-x channel? Please confirm by saying "Yes".'
      }
```